### DISCUSSION TITLE
Show fields from Accounts to Project

### DISCUSSION THREAD

**Discussion by Andr198** [YetiForceCompany/YetiForceCRM#14965] [How to]
Нow to show fields from accounts to a project?
Is it possible to do this with Widgets?

**Comment by mariuszkrzaczkowski** [comment]
where on the list or record details?

**Comment by Andr198** [comment]
Summary and details

**Reply by rskrzypczak** [reply]
The easiest way is to add them to the header of the card.
This option is available only when the reference field is associated with only one module, so if you are not using the Vendor module, disable this module, and then in the field configuration it will be able to choose which field you want to display.
![image](https://user-images.githubusercontent.com/10221999/125895168-9bc1a823-d4fa-4f4d-83ea-ab60ad74dbc9.png)

**Reply by Andr198** [reply]
Thank you. Useful function.
But only there is a limit of 3 item
I have more fields and some of them Dynamic Dependent Fields

**Reply by mariuszkrzaczkowski** [reply]
you can also try the 'Process wizard' tab
https://gitdeveloper.yetiforce.com/index.php?module=SSalesProcesses&view=Detail&record=496&parent=118&mid=119&mode=processWizard&tab_label=LBL_RECORD_PROCESS_WIZARD

![image](https://user-images.githubusercontent.com/10198654/125922886-ed2fd858-1945-4750-9160-1d4b8699f312.png)

hide buttons and progress bar, here you can show related fields
only this requires creating a file in a module and knowledge of at least basic programming

**Reply by Andr198** [reply]
How to activate it?
In the downloaded version I do not have this

**Comment by Andr198** [comment]
How to find file modules / SSalesProcesses / models / ProcessWizard.php ???
Needed for example.

**Reply by mariuszkrzaczkowski** [reply]
https://github.com/YetiForceCompany/YetiForceCRM/discussions/15114
